robocorp-code/src/robocorp_code/robocorp_language_server.py

```diff
@@ -259,6 +259,25 @@ def _cloud_login(self, params: CloudLoginParamsDict) -> ActionResultDict:
             result = self._rcc.credentials_valid()
         return {"success": result, "message": None, "result": result}
 
+    @command_dispatcher(commands.ROBOCORP_CLOUD_LOGOUT_INTERNAL)
+    def _cloud_logout(self) -> ActionResultDict:
+        from robocorp_ls_core.progress_report import progress_context
+
+        self._feedback_metric("vscode.cloud.logout")
+
+        # When credentials are removed we need to remove existing caches.
+        self._dir_cache.discard(self.CLOUD_LIST_WORKSPACE_CACHE_KEY)
+
+        with progress_context(
+                self._endpoint, "Removing cloud credentials", self._dir_cache
```

Comment:
This formatting seems a bit strange... Please use the `black` formatter to format code in your IDE.

Reply:
reformatted the code, and also ran through black